tests/Feature/RegisterTest.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace Uasoft\Badaso\Module\LMS\Tests\Feature;
+
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Str;
+use Tests\TestCase;
+use Uasoft\Badaso\Helpers\CallHelperTest;
+use Uasoft\Badaso\Module\LMSModule\Models\User;
+
+class RegisterTest extends TestCase
+{
+    public function testStartInit()
+    {
+        CallHelperTest::handleUserAdminAuthorize($this);
+    }
+
+    public function testAddUserDatabase()
+    {
+        $name = Str::random(10);
+        $create_user = [
+            'name' => $name,
+            'username' => $name,
+            'email' => $name.'@gmail.com',
+            'password' => Hash::make($name),
+        ];
+        $user = User::create($create_user);
+        $this->assertEquals($create_user['name'], $user->name);
+        $this->assertEquals($create_user['username'], $user->username);
+        $this->assertEquals($create_user['email'], $user->email);
+        $this->assertEquals($create_user['password'], $user->password);
+
+        $user->delete();
```

Comment:
no need to delete it. Don't worry about the database since it's always getting refreshed each time we run a test.